src/datreant/core/agglimbs.py

```diff
@@ -278,6 +278,22 @@ def fuzzy(self, tag, threshold=80, scope='all'):
 
         return tuple(matches)
 
+    def filter(self, tag):
+        """Given tag(s), get a Bundle of Treants whose tags match it.
```

Comment:
Probably should be written as, "Filter out a Bundle of members matching the given tag expression."